### ISSUE TITLE
Improve itinerary returned and trip planner widgets

### ISSUE THREAD

**Issue by jailby** [CUTR-at-USF/usf-mobullity#120]
I personally think that having to separate widget for "itinerary returned" and "trip planner" is not the best solution in term of usability.

It is not easy for a user to switch options in the trip planner if the itinerary return is not convient for him/her.

Other open trip planner fork handle this situation differently.

For example, [this one](http://opentripplanner.usf.edu/index.html#/submit&fromPlace=ENB::28.058700,-82.415200&toPlace=LIB::28.059700,-82.412200&mode=WALK&min=QUICK&maxWalkDistance=840&walkSpeed=1.341&time=10:21am&date=11/6/2015&arriveBy=false&itinID=1&wheelchair=false&preferredRoutes=&unpreferredRoutes=) and [this one](http://ride.trimet.org/#/submit&fromPlace=SW Vermont Ct @ SW Vermont St::45.476480,-122.747350&toPlace=SW 25th St @ SW 25th Ct::45.478783,-122.467514&mode=TRANSIT,WALK&min=QUICK&maxWalkDistance=840&time=10:35 am&date=11/6/2015&arriveBy=false&itinID=1&wheelchair=&preferredRoutes=&unpreferredRoutes=) have both widgets in a same "parent" widget but within two separate "tabs".

Another solution for us could be to return the itinerary within the same widget but at the bottom under the button "Plan your trip". Currently this area of the widget is only fill up with useless whit space.

This issue was originally brought up after a discussion on it and create in #35 but this is a personal opinion and maybe I'm wrong on the fact that the current state with two widgets is usable.

**Comment by barbeau** [comment]
Yeah, I've never liked the whole separate floating widgets thing.  I'd vote for a change where both sets of info are managed within a single window.  I'm not sure if the tabs solution is best either, though - I've never thought that opening new results in separate tabs was very intuitive.  We should collect some screenshots of other native/web apps to see how they are handled.
